notebook/templates/page.html

```diff
@@ -98,8 +98,16 @@
           }
       })
 
+    // Detect Safari
+    let chromeAgent = navigator.userAgent.indexOf("Chrome") > -1;
+    let safariAgent = navigator.userAgent.indexOf("Safari") > -1;
+    if (chromeAgent && safariAgent) safariAgent = false;
```

Comment:
Ah, I think this needs to be put into a block with `{ }`

Reply:
Please check again whether the new change is what you want.